scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Job.java

```diff
@@ -335,6 +353,17 @@ public static Job parseJSONInfo(JSONObject jsonJobInfo) {
         long startTime = (long) jsonJobInfo.get("startTime").isNumber().doubleValue();
         long inErrorTime = (long) jsonJobInfo.get("inErrorTime").isNumber().doubleValue();
         long finishedTime = (long) jsonJobInfo.get("finishedTime").isNumber().doubleValue();
+        Map<String, String> genericInformation = new HashMap<>();
```

Comment:
Can you refactor this logic in a private method that could be called extractGenericInformation(jsonJobInfo) returning the Map of the generic information